src/oaklib/converters/owl_to_fhir_codesystem.py

```diff
@@ -0,0 +1,233 @@
+      - title
+      - version
+      - version_iri
+    - stuff from aherc/fhir-owl (some are duplicates of above)
+      "resourceType": "CodeSystem",
+      "url": "http://purl.obolibrary.org/obo/mondo.owl",
+      "version": "http://purl.obolibrary.org/obo/mondo/releases/2022-07-01/mondo.owl",
+      "name": "http://purl.obolibrary.org/obo/mondo.owl",
+      "status": "draft",
+      "experimental": false,
+      "description": "Includes Ontology(OntologyID(Anonymous-51)) [Axioms: 79310 Logical Axioms: 0]",
+      "valueSet": "http://purl.obolibrary.org/obo/mondo.owl?vs",
+      "hierarchyMeaning": "is-a",
+      "compositional": false,
+      "versionNeeded": false,
+      "content": "complete",
+      "count": 41193,
+  2. Add converter to CLI
+  3. Test(s)
+  4. Improve converter
+
+TODO's expanded
+  4. Improve converter
+  Paths to things to use (relative to root of Graph `g`):
+  4.1. Obographs stuff
+  4.1.1. propertyChainAxioms: understand & utilize
+    "propertyChainAxioms" : [ {
+      "predicateId" : "http://purl.obolibrary.org/obo/RO_0002566",
+      "chainPredicateIds" : [ "http://purl.obolibrary.org/obo/RO_0002327", "http://purl.obolibrary.org/obo/RO_0002411", "http://purl.obolibrary.org/obo/RO_0002233" ]
+    }, ...
+  4.1.2. domainRangeAxioms: understand & utilize
+    "domainRangeAxioms" : [ {
+      "predicateId" : "http://purl.obolibrary.org/obo/RO_0002176",
+      "domainClassIds" : [ "http://purl.obolibrary.org/obo/BFO_0000004" ]
+    }, ...
+  4.1.3. domainRangeAxioms: understand & utilize
+    "logicalDefinitionAxioms" : [ {
+      "definedClassId" : "http://purl.obolibrary.org/obo/MONDO_0011258",
+      "genusIds" : [ "http://purl.obolibrary.org/obo/MONDO_0018878" ],
+      "restrictions" : [ {
+        "propertyId" : "http://purl.obolibrary.org/obo/RO_0004003",
+        "fillerId" : "http://identifiers.org/hgnc/3519"
+      } ]
+    }, ...
+  4.1.4. meta: understand & utilize
+    "meta" : {
+      "subsets" : [ ],
+      "xrefs" : [ ],
+      "version": ...,
+      "basicPropertyValues" : [ {
+        "pred" : "http://purl.obolibrary.org/obo/IAO_0000700",
+        "val" : "http://purl.obolibrary.org/obo/MONDO_0000001"
+      }, ...
+  4.1.5. equivalentNodesSets: understand & utilize?
+    "equivalentNodesSets" : [ ],  # empty for Mondo
+  4.2. aehrc/fhir-owl stuff
+  4.2.1. filter[]: Look at the file I converted using fhir-owl, near the top.
+  4.3. Obograph Node stuff
+  4.3.1. See what all I can include from:
+    - alternative_ids[], created_by, creation_date, definition, definition_provenance, id, lbl (label), synonyms[],
+    - xrefs[] (I see the CURIE, but nothing else... does it not parse other xref info? or is it in axioms?),
+    - meta.version?
+  4.4. Anything else from FHIR CodeSystem?
+
+todo's (minor/later)
+  1. bioontologies -> OAK when issue fixed: https://github.com/INCATools/ontology-access-kit/issues/321
+  2. Schema mappings (Obograph -> FHIR CodeSystem). SchemaAutomator? Other approach?
```

Comment:
not so relevant here

Reply:
Perhaps so, and I don't want to do it (at least not now) either. However, I do think to myself why not. If I do formal schema mappings for, say, (a) `LinkML Enum`::`FHIR ValueSet` and (b) `SSSOM`::`FHIR ConceptMap`, why not (c) `Obograph`::`FHIR CodeSystem`? In all cases, there are formal specifications. So I am wondering why (c) would be the odd one out in this case?

Reply:
I think you're right!